### ISSUE TITLE
False positive for content -> contents (218)

### ISSUE THREAD

**Issue by hafkensite** [warpnet/salt-lint#252]
**Describe the bug**
The new [TypoContentsRule](https://github.com/warpnet/salt-lint/blob/v0.6.0/saltlint/rules/TypoContentsRule.py) rule assumes all instances of 'content' should be 'content**s**', but for `file.line` the parameter is actually called 'content'. 

**To Reproduce**
Run version 0.6.0 on code like this:
```yaml
gitlab-runner-listen-address:
  file.line:
    - name: /etc/gitlab-runner/config.toml
    - content: 'listen_address = "0.0.0.0:9252"'
    - match: '^listen_address = .*'
```

**Expected behavior**
The 218 rule only matches cases where its actually a typo.

**Desktop (please complete the following information):**
- OS: Arch
- Version 0.6.0

**Comment by hbokh** [comment]
Can confirm - with output regarding @hafkensite his lines:

```
$ salt-lint salt/gitlab-runner/init.sls
Examining salt/gitlab-runner/init.sls of type sls
[218] "content" looks like a typo. Did you mean "contents"?
salt/gitlab-runner/init.sls:49
    - content: 'listen_address = "0.0.0.0:9252"'
```

**Comment by jbouter** [comment]
@roaldnefs @jay7x I suggest we change the typo-catching rule to only work on states we have verified.

**Comment by jbouter** [comment]
Fixed in #253 and removed in the latest release (`v0.6.1`). Thanks for alerting us @hafkensite and @hbokh! 

Once we've got the time, we'll work on implementing a better check that targets only specific states, in which we know `content` will fail and `contents` should be set.

**Comment by jay7x** [comment]
Thank you for fixing this.. I didn't know there is a case when `content` is valid.. What I hate in SaltStack is inconsistency :(
